yalc.lock

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "v1",
+  "packages": {
+    "@stripe/stripe-react-native": {
+      "signature": "ab0fb0b56035d4b459834a1bd9486b72",
+      "file": true
+    }
+  }
+}
```

Comment:
You have to make yalc use not requiring to install it globally: 
+ yalc must be a ev-mobile development package;
+ ad hoc npm script must be created to build, install stripe package and yalc local repo with it inside. 

e-mobility repos must be self-contained and not require any global packages install to work.   

Or if you wait for the official release, you have to make sure it will be done on time.

Reply:
If I create a script, it will require yalc. The first command is yalc add @stripe....
Also, where should I put the script ?

Reply:
Yes it's done